### PR TITLE
Term length

### PR DESCRIPTION
**Pull request by minhbsq**

### CODE REVIEW

File: cluster/options.go [hunk 1]
```diff
@@ -30,15 +30,15 @@ type Options struct {
 
 func NewOptions() *Options {
 	archiveOpts := archive.DefaultOptions()
-	archiveOpts.RequestChannel = "aeron:ipc?alias=cluster-service-archive-ctrl-req|term-length=128k"
-	archiveOpts.ResponseChannel = "aeron:ipc?alias=cluster-service-archive-ctrl-resp|term-length=128k"
+	archiveOpts.RequestChannel = "aeron:ipc?alias=cluster-service-archive-ctrl-req|term-length=2097152"
+	archiveOpts.ResponseChannel = "aeron:ipc?alias=cluster-service-archive-ctrl-resp|term-length=2097152"
```
Comment (Aaronong) [inline cluster/options.go:34]: No let's not do it here.

Do it in the platform lib instead

**Review by Aaronong — CHANGES_REQUESTED** [review]
Do in platform lib

**Review by Aaronong — CHANGES_REQUESTED** [review]
^

**Comment by minhbsq** [comment]
Oh  I see. In fact David has already changed it in platform lib, but some how it's not applied

**Pull request closed without merging**